ui/app/rif/components/luminoNetworkChannels.js

```diff
@@ -0,0 +1,124 @@
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import {CustomButton, GenericTable} from './index';
+import rifActions from '../actions';
+import ItemWithActions from './item-with-actions';
+import {faCoins} from '@fortawesome/free-solid-svg-icons';
+import { SVG_PLUS } from '../constants';
+
+class LuminoNetworkChannels extends Component {
+
+  static propTypes = {
+    pageName: PropTypes.string.isRequired,
+    isOwner: PropTypes.bool,
+    paginationSize: PropTypes.number,
+    getChannelsGroupedByNetwork: PropTypes.func,
+    getChannel: PropTypes.func,
+    classes: PropTypes.any,
+  }
+
+  constructor (props) {
+    super(props);
+    this.state = {
+      networkChannels: [],
+    };
+  }
+
+  getData () {
+    const {classes} = this.props;
+    if (this.state.networkChannels) {
+      return this.state.networkChannels.map(networkChannel => {
+        const item = <ItemWithActions leftIcon={{icon: faCoins, color: '#05836D'}} contentClasses={classes.content} actionClasses={classes.contentActions}
+                                      text={networkChannel.token_symbol} enableRightChevron={true}/>
+        return {
+          content: item,
+        }
+      });
+    }
+    return [];
+  }
+
+  componentDidMount () {
+    this.loadChannels();
+  }
+
+  loadChannels () {
+    this.props.getChannelsGroupedByNetwork().then(networkChannels => {
+      const arrayNetworks = [];
+      for (let key of Object.keys(networkChannels)) {
```

Comment:
Variable key doesn't change, it can be a const

Reply:
Changed to be const